src/Components/Header/Header.js

```diff
@@ -12,12 +12,12 @@ const Header = () => {
         <NavLink className="nav-items" to="/about">
           about
         </NavLink>
-        <NavLink className="nav-items" to="/contact">
-          contact
-        </NavLink>
         <NavLink className="nav-items" to="/styles">
           styles
         </NavLink>
+        <NavLink className="nav-items" to="/contact">
+          contact
+        </NavLink>
```

Comment:
Contacting someone is usually the last thing people want to do on a site, so I moved the contact form link accordingly.